lib/tasks/sitemap_robots.rake

```diff
@@ -0,0 +1,75 @@
+namespace :sitemap do
+  desc 'Generate robots.txt including sitemap link'
+  task robots: :environment do
+    # Use a dynamic sitemap URL based on environment: production or localhost (need staging?)
+
+    default_host = {
+      'production' => 'https://geodata.lib.berkeley.edu',
+      'staging' => 'https://geodata.ucblib.org',
+      'development' => 'http://localhost:3000'
+    }[Rails.env] || 'https://geodata.ucblib.org'
```

Comment:
Same issue as above RE: how RAILS_ENV is set / used.

Reply:
and agreed here - i prefer the strategy that @danschmidt5189 proposed.